h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderBuilder.java

```diff
@@ -54,10 +59,8 @@ public void computeImpl() {
 
       disableIgnoreConstColsFeature();
       TargetEncoderModel.TargetEncoderOutput output = new TargetEncoderModel.TargetEncoderOutput(TargetEncoderBuilder.this, _targetEncodingMap, priorMean);
-      _targetEncoderModel = new TargetEncoderModel(_job._result, _parms, output, tec);
-
-      _targetEncoderModel.write_lock(_job);
-      _targetEncoderModel.unlock(_job);
+      _targetEncoderModel = new TargetEncoderModel(_result, _parms, output, tec);
```

Comment:
``` job._result``` was a wrong key to pass to a Model. It was causing key to be removed by Scope. Needed key that is stored in `ModelBuilder`'s `_result` field first, matches keys that are used to retrieve models from the grid ( this way we don't get N null models in the end) and second, this key is being created outside of (before) Scope.enter() and is not being unnecessary deleted from DKV.